application/src/main/java/org/thingsboard/server/service/ws/telemetry/cmd/TelemetryCmdsWrapper.java

```diff
@@ -43,6 +44,8 @@
 @Deprecated
 public class TelemetryCmdsWrapper {
 
+    private List<RpcCmd> rpcCmds;
```

Comment:
This commands wrapper is deprecated, so don't add new types of commands here.

Reply:
There are many commands here now, but not many in the future file, so should we put them here first?